apps/site/assets/ts/schedule/components/ScheduleLoader.tsx

```diff
@@ -220,7 +223,9 @@ export const ScheduleLoader = ({
     }
 
     if (component === "SCHEDULE_FINDER") {
-      return (
+      return isFerryRoute ? (
```

Comment:
Why not put this in with the if clause above (e.g `if (component === "SCHEDULE_FINDER" && !isFerryRoute)`?